### PR TITLE
Feature/tests

### PR DESCRIPTION
**Pull request by ashwch**

### CODE REVIEW

File: tests/payloads/payment_requests.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    'payment_request_create': {
+        'request': {
+            'purpose': 'FIFA 16',
+            'amount': '2500',
+            'buyer_name': 'Jon Snow',
+            'email': 'jon+snow@mailinator.com',
+            'phone': '+919999999990',
+            'send_email': True,
+            'send_sms': True,
+            'redirect_url': 'https://www.instamojo.com/redirect_handler',
```
Comment (indradhanush) [inline tests/payloads/payment_requests.py]: Lets use hyphenated urls?

File: tests/payloads/payment_requests.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            'webhook': 'https://www.instamojo.com/webhook_handler',
```
Comment (indradhanush) [inline tests/payloads/payment_requests.py]: This as well

File: tests/test_payment_requests.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
+from unittest import TestCase
+
+import responses
+
+from instamojo_wrapper import Instamojo
+from tests.payloads.payment_requests import payment_requests_payload
```
Comment (indradhanush) [inline tests/test_payment_requests.py]: Do you want the payloads to be available at `tests.payloads` instead ?

File: tests/test_products.py [hunk 4]
```diff
@@ -0,0 +1,250 @@
+from unittest import TestCase
+
+import responses
+from mock import mock_open, patch
+
+from instamojo_wrapper import Instamojo
+from tests.payloads.products import products_payload
+
+
+class Testproducts_payload(TestCase):
```
Comment (indradhanush) [inline tests/test_products.py]: Underscored? o_0
And why not `TestProducts` ?

File: tests/test_refunds.py [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+            body='{}',
+            content_type='application/json'
+        )
+        resp = self.api.refunds_list(**data['request'])
+        self.assertEqual(resp, {})
+        self.assertEqual(len(responses.calls), 1)
+        self.assertEqual(
+            responses.calls[0].request.url, endpoint)
+
+
```
Comment (indradhanush) [inline tests/test_refunds.py]: Blank lines.

File: tests/payloads/products.py [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+    },
+    'product_details': {
+        'request': {
+            'slug': 'hello-api'
+        }
+    },
+    'products_list': {
+        'request': {}
+    }
+}
```
Comment (indradhanush) [inline tests/payloads/products.py]: No newline.

**Comment by indradhanush** [comment]
You might want to add tests for the `tests.payloads` packages as well.

File: tests/test_products.py (hunk 4 above)
Comment (ashwch) [inline tests/test_products.py]: Victim of mass copy-replace. :-/

**Pull request merged**